src/workers/deployment-image-update/index.js

```diff
@@ -0,0 +1,85 @@
+import "dotenv/config";
+import { prisma } from "generated/client";
+import commander from "commander";
+import log from "logger";
+import { generateNamespace } from "deployments/naming";
+import { generateHelmValues } from "deployments/config";
+import { natsPubSub } from "nats-streaming";
+import { DEPLOYMENT_AIRFLOW, DEPLOYMENT_IMAGE_UPDATED } from "constants";
+
+const nc = createNatsClient();
+
+/**
+ * NATS Deployment Update Worker
+ */
+function createNatsClient() {
+  const clientID = "deployment-image-update-worker";
+  const subject = DEPLOYMENT_IMAGE_UPDATED;
+  const nc = natsPubSub(clientID, subject, helmUpdateDeployment);
+  log.info(`NATS ${clientID} Running...`);
+
+  return nc;
+}
```

Comment:
```suggestion
const nc = natsPubSub("deployment-image-update-worker", DEPLOYMENT_IMAGE_UPDATED, helmUpdateDeployment);
log.info(`NATS ${clientID} Running...`);
```

Now that I look at this again, could we shorten it down to this?

Reply:
Yeah, that'd work.  There is error handling using `nc.on("error", ...` so we don't actually need that in a try/catch like I originally though.  I kind of like the idea of having:

```
const CLIENT_ID = "deployment-image-update-worker";
const nc = natsPubSub(CLIENT_ID, DEPLOYMENT_IMAGE_UPDATED, helmUpdateDeployment);
log.info(`NATS ${clientID} Running...`);
```

Does that seem all right, @schnie?

Reply:
>const CLIENT_ID = "deployment-image-update-worker";
let's move to constants.js or so

Reply:
the `const CLIENT_ID` is different for every worker.  We can probably move to constants, but we'll need to change the name for each new worker.